readme.md

```diff
@@ -1 +1,60 @@
-# Hello World
+# Form Builder
+
+**Form Builder** is a simple node application that generates a form based on given inputs.
+
+<div align="center">
+
+​		<img src="/docs/demo.png" width="220">
```

Comment:
Check out the explanation [here on how to render images](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) within a markdown file.